addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -113,6 +113,16 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * Last visible message is scoped to the thread cache, it's needed to be reseted when
```

Comment:
- Split "what" and "why" in 2 sentences. Basically replace comma by dot.
- _"reseted"_ => _"reset"_